VAMobile/src/components/CommonErrorComponents/DowntimeError.tsx

```diff
@@ -3,22 +3,21 @@ import { useTranslation } from 'react-i18next'
 import React, { FC } from 'react'
 
 import { AlertBox, Box, VAScrollView } from 'components'
-import { DowntimeScreenIDToFeature, ScreenIDTypes } from 'store/api/types'
 import { ErrorsState } from 'store/slices'
 import { NAMESPACE } from 'constants/namespaces'
 import { RootState } from 'store'
+import { ScreenIDToDowntimeFeature, ScreenIDTypes, ScreenNameFeatureConstants } from 'store/api/types'
 import { useSelector } from 'react-redux'
 import { useTheme } from 'utils/hooks'
 
+
 export type DowntimeErrorProps = {
   /**The screen id for the screen that has the errors*/
   screenID: ScreenIDTypes
-  /**Override the feature name in the event that a feature happens to share the same api error(ex:contact information and personal information) */
-  overrideFeatureName?: string
```

Comment:
Removing this because I'm changing the pattern to be driven by screen id, which *should* be unique, which I believe solves the problem this was intended to fix. In theory. Plausible we could want more complex logic in the future.